src/v2/client/sessionStorageHelper.js

```diff
@@ -11,19 +11,36 @@
  */
 
 const STATE_HANDLE_SESSION_STORAGE_KEY = 'osw-oie-state-handle';
+const APP_ID_SESSION_STORAGE_KEY = 'osw-oie-app-id';
+
+const getSessionStorageKey = (appId) => {
+  return appId ? `${STATE_HANDLE_SESSION_STORAGE_KEY}-${appId}` : STATE_HANDLE_SESSION_STORAGE_KEY;
```

Comment:
why we are still appending the `appId` to key name ? It's not relevant anymore right ?